frontend/src/routes/Upload.tsx

```diff
@@ -157,6 +162,21 @@ const CancelButton: React.FC<CancelButtonProps> = ({ abortController }) => {
     );
 };
 
+const createPlaceholderMutation = graphql`
+    mutation UploadCreatePlaceholderMutation($event: NewEvent!) {
+        createPlaceholderEvent(event: $event) { id }
+    }
+`;
+
+type PlaceholderEvent = {
+    opencastId: string;
+    title: string;
+    description?: string;
+    seriesId?: string;
+    creators: string[];
+    acl: Acl;
+}
```

Comment:
Can this be imported from the autogenerated TS file? I feel like this type definition should be in there somewhere...